icekit/project/settings/_base.py

```diff
@@ -580,6 +580,7 @@
     'icekit.response_pages',
     'notifications',
 
+    'icekit.authors',
```

Comment:
Agreed on option 2. At the moment, the Author functionality is inextricable from a listing page to mount the URL under.